web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -64,7 +64,7 @@ angular.module('oncokbApp')
                         if(scope.rs && _.isNull(scope.rs.get('lastReviewed')) && scope.rs.get('action') !== 'rejected') {
                             scope.rs.set('lastReviewed', o);
                         }
-                        scope.content.stringO = stringUtils.getTextString(scope.object.text);
+                        scope.content.stringO = scope.object.text;
```

Comment:
yeah I made that fix for the text style issue. but That's not right fix. The real fix should be using ng-model for display instead of angular expression